### PR TITLE
Clean up incosistent hack to patch document-register-element.

### PR DESCRIPTION
**Pull request by cramforce**
Now uses the patched file everywhere (instead of different local/testing/compiled code path) and avoids relying on closure compiler file shadowing (by renaming the file instead of copying to shadow dir).

### CODE REVIEW

**Comment by cramforce** [comment]
CC @prateekbh This should make it easier to avoid the polyfill altogether.

File: build-system/tasks/update-packages.js [hunk 1]
```diff
@@ -55,6 +55,30 @@ function patchWebAnimations() {
...
+        'node_modules/document-register-element/build/' +
+        'document-register-element.node.js').toString();
+    // Eliminate the immediate side effect.
+    file = file.replace('installCustomElements(global);', '');
+    // Closure Compiler does not generate a `default` property even though
+    // to interop CommonJS and ES6 modules. This is the same issue typescript
+    // ran into here https://github.com/Microsoft/TypeScript/issues/2719
+    file = file.replace('module.exports = installCustomElements;',
+        'module.exports = exports.default = installCustomElements;');
+    fs.writeFileSync(patchedName, file);
```
Comment (rsimha) [inline build-system/tasks/update-packages.js]: Add a logging statement for consistency:
```
if (!process.env.TRAVIS) {
  log(colors.green('Patched'), colors.cyan(patchedName));
}
```

Comment (cramforce) [reply build-system/tasks/update-packages.js]: Done.

**Comment by cramforce** [comment]
Part of #16847

**Pull request merged**